modules/oe_content_event/oe_content_event.post_update.php

```diff
@@ -133,3 +133,20 @@ function oe_content_event_post_update_20003(): void {
   $field_config->setSetting('title', 2);
   $field_config->save();
 }
+
+/**
+ * Set "composite revisions" option for reference fields.
+ */
+function oe_content_event_post_update_20004(): void {
+  $fields = [
+    'node.oe_event.oe_event_contact' => TRUE,
+    'node.oe_event.oe_event_media' => FALSE,
+    'node.oe_event.oe_event_venue' => TRUE,
+    'oe_event_speaker.oe_default.oe_person' => FALSE,
```

Comment:
This config belongs to the submodule providing the EventSpeaker subentity, so I think it should be updated in a post_update of the submodule. The submodule is not a dependency of the main module so the config might not event exist in some cases.